### ISSUE TITLE
Allow custom sort order for environments

### ISSUE THREAD

**Issue by rm-code** [usebruno/bruno#2096]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Currently environments are sorted alphabetically. 

![image](https://github.com/usebruno/bruno/assets/11627131/8710ae70-0d13-4a5f-8c5a-967ead34906b)

A custom sort order would be a nice addition, especially when you are working with a lot of different environments. Current workaround is using a numbered prefix.

### Mockups or Images of the feature

The nicest way would be to adopt the drag-and-drop reordering we already have in place for requests. Each environment would then have to store its sequence. As far as I can tell, this wouldn't be easy to do with the current key-value setup the environments have, but maybe it could be implemented together with #292?

**Comment by maisgold** [comment]
that would be a nice feature. I use Bruno with over 30 global environements.  THX.

**Comment by dthomas-oddball** [comment]
yes please add this
~~at minimum sort it alphabetically so we can name them 1 2 3 4 if we have to~~

**Comment by markalroberts** [comment]
Yes please! I find myself having to delete all the environments and re-add in exactly the right order I want them to show on the screen - it's very brittle and a pain if using lots of repos for lots of web apps as we do!
